src/Uno.Extensions.Maui.UI/Extensibility/IMauiEmbeddingExtension.cs

```diff
@@ -18,22 +18,22 @@ public interface IMauiEmbeddingExtension
 	/// Initializes the Maui embedding extension with the specified application.  
 	/// </summary>  
 	/// <param name="iApp">The application to initialize.</param>  
-	void Initialize(IApplication iApp) => throw new PlatformNotSupportedException();
+	void Initialize(IApplication iApp) { }
```

Comment:
Maybe best to add a comment to clarify method that intentionally empty like this one for example?